<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Расчет третьей стороны треугольника</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="ab_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>AB=</string>
    </property>
   </widget>
   <widget class="QLabel" name="bc_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>BC=</string>
    </property>
   </widget>
   <widget class="QLabel" name="y_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>y=</string>
    </property>
   </widget>
   <widget class="QLabel" name="ac_label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>AC=</string>
    </property>
   </widget>
   <widget class="QLabel" name="ac_num">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>num1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Act_Button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ab_SpinBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="bc_SpinBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="y_SpinBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Угол измеряется в</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rad_radioButton">
        <property name="text">
         <string>радианах</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="deg_radioButton">
        <property name="text">
         <string>градусах</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
